src/host/services/NMEA2000Service.cpp

```diff
@@ -61,14 +61,14 @@ void NMEA2000Service::setup() {
 
 void NMEA2000Service::sendN2kMessage(const tN2kMsg& msg) {
   bool result = NMEA2000.SendMsg(msg);
-
+  /*
   DEBUG("Sending message on n2k bus - pgn=%i prio=%i src=%i dst=%i len=%i result=%s", msg.PGN, msg.Priority,
       msg.Source,
       msg.Destination, msg.DataLen, result ? "success":"fail");
-
-  char pcdin[100];
-  N2kToSeasmart(msg, now(), pcdin, sizeof(pcdin));
-  DEBUG("TX: %s", pcdin);
+  */
+  //char pcdin[100];
```

Comment:
This is an example of a message that should not be here anymore. Please remove it completely. Sorry about that!